README.md

```diff
@@ -17,16 +17,22 @@ Result: `E-TEST-1: Something went wrong.`
 You can specify placeholders in the message and fill them up with parameters values:
 
 ```go
-	renderedString := error_reporting_go.ExaError("E-TEST-2").Message("Unknown input {{input}}.").Parameter("input", "unknown").String()
+renderedString := error_reporting_go.ExaError("E-TEST-2").Message("Unknown input {{input}}.").Parameter("input", "unknown").String()
 ```
 
 Result: `E-TEST-2: Unknown input 'unknown'.`
 
 `ParameterWithDescription(name, value, description)` method also gets a parameter description which will be used to generate a parameter description for the error-catalog.
 
-The builder automatically quotes parameters (depending on the type of the parameter) with single quotes.
+The builder automatically quotes parameters with single quotes.
```

Comment:
In Java, we quote depending on the type. So we don't quote numbers for example.

Reply:
This is not supported here currently but can be considered as a new feature